nix/modules/flake-module.nix

```diff
@@ -75,12 +75,21 @@
                 AllowTcpForwarding no
             '';
         };
-        system.activationScripts.set-jail = ''
-            ${pkgs.coreutils}/bin/chown root:root /home/jail
-            ${pkgs.coreutils}/bin/mkdir -p /home/jail/run/current-system/sw
-            ${pkgs.coreutils}/bin/cp -r ${pkgs.bash}/bin /home/jail/run/current-system/sw
-            ${pkgs.nix} copy --from / --to /home/jail ${pkgs.bash}/bin/bash
-        '';
+  systemd.services.set-jail = {
+    wantedBy = [ "multi-user.target" ];
+    path = [ config.nix.package ];
+    script = ''
+      chown root:root /home/jail
+      mkdir -p /home/jail/run/current-system/sw/bin
+      ln -s ${pkgs.bash}/bin/bash /home/jail/run/current-system/sw/bin/bash
+      nix copy --from / --to /home/jail ${pkgs.bash}/bin/bash
```

Comment:
The nix command is absent with the following log, then I try to use `pkgs.nix` on next commit
```
Aug 29 08:29:02 kld-10 systemd[1]: Starting set-jail.service...
Aug 29 08:29:02 kld-10 set-jail-start[934]: /nix/store/n9by8r8ddz7r5hprd1g8sc7ajjbc5sd2-unit-script-set-jail-start/bin/set-jail-start: line 6: nix: command not found
Aug 29 08:29:02 kld-10 systemd[1]: set-jail.service: Main process exited, code=exited, status=127/n/a
Aug 29 08:29:02 kld-10 systemd[1]: set-jail.service: Failed with result 'exit-code'.
Aug 29 08:29:02 kld-10 systemd[1]: Failed to start set-jail.service.
```